src/v/resource_mgmt/storage.cc

```diff
@@ -206,13 +206,16 @@ ss::future<> disk_space_manager::manage_data_disk(uint64_t target_size) {
                                  ? 0
                                  : usage.usage.total() - target_size;
     if (target_excess <= 0) {
-        vlog(
-          rlog.info,
+        vlogl(
+          rlog,
+          _previous_reclaim ? ss::log_level::info : ss::log_level::debug,
           "Log storage usage {} <= target size {}. No work to do.",
           human::bytes(usage.usage.total()),
           human::bytes(target_size));
+        _previous_reclaim = false;
         co_return;
     }
+    _previous_reclaim = true;
```

Comment:
we could do that, are you thinking like a log throttling mechanism? this mechanism prints at info level if we had previously been reclaiming, otherwise we were already looking good in the previous loop so we print at debug level.

Reply:
ok, now i think i get it how it's supposed to work. i thought it would just ping pong between debug and info, but i see that it's meant to print only the first message as info and the rest will be at debug until conditions change